libs/wingsdk/src/core/resource.ts

```diff
@@ -106,6 +112,8 @@ export abstract class Resource
     });
   }
 
+  private readonly bindMap: Map<IResource, Set<string>> = new Map();
```

Comment:
Not sure if this will be used for any different binding in the future, but for now we should limit the typing?

```suggestion
  private readonly bindMap: Map<IInflightHost, Set<string>> = new Map();
```

Reply:
Good idea